README.md

```diff
@@ -22,18 +22,29 @@ These instructions illustrate deployment to a Kubernetes cluster on AWS.
 
 Please check the Anaxes Shipyard documentation on [running a cluster](https://github.com/Alfresco/alfresco-anaxes-shipyard/blob/master/SECRETS.md).
 
-If you are deploying the Identity Service into a cluster with other Alfresco components such as Content Services and Process Services, a VPC and cluster with 5 nodes is recommended. Each node should be a m4.xlarge EC2 instance.
+Note the resource requirements:
+
+* Minikube: At least 2 gigs of memory, i.e.:
```

Comment:
Minor: as we're systematically stripping minikube references from other documents, maybe we should do it here too. I appreciate this is outside the scope of this ticket.